src/nidcpower/metadata/functions.py

```diff
@@ -3496,11 +3496,19 @@
         'returns': 'ViStatus'
     },
     'SetRuntimeEnvironment': {
-        'codegen_method': 'library-only',
+        'codegen_method': 'private',
```

Comment:
I'm guessing 'private' or 'public' here makes no difference since you aren't rendering in `Session`? Maybe add a comment.

Reply:
Added an actual comment in the functions_addon override. The same comment will be added to our internal metadata, but it won't make it into the exported metadata.